README.md

```diff
@@ -24,6 +24,28 @@ This library requires:
 
 This package provides a Nuke plugin that creates jobs for AWS Deadline Cloud using the [AWS Deadline Cloud client library][deadline-cloud-client]. Based on the loaded comp it determines the files required, allows the user to specify render options, and builds an [OpenJD template][openjd] that defines the workflow.
 
+### Getting Started
+
+To install the submitter manually, you can use pip.
+
+```sh
+$ pip install deadline-cloud-for-nuke -t target-directory
```

Comment:
Nit: It'd be slightly better to choose a real target directory like `~/deadline-cloud-for-nuke` so that users can copy-paste the commands. Then you could also note that users can choose a different directory if they want.

Reply:
Yea, we'd then also remove the shell indicator `$` (for lack of knowing the term) to allow the copy/paste.